projects/win32/f3d.bundle.cmake

```diff
@@ -31,6 +31,14 @@ if (ospray_enabled)
   endforeach ()
 endif ()
 
+# OpenUSD plugin descriptors
+if (openusd_enabled)
+  install(
+    DIRECTORY   "${superbuild_install_location}/lib/usd"
+    DESTINATION "lib"
+    USE_SOURCE_PERMISSIONS)
+endif()
```

Comment:
if this is the same logic with different destination, maybe put in common ? See how its done for the licenses.

Reply:
I tried that but for some reason it's failing, I'm not sure why yet.

Reply:
wants me to take a look ?

Reply:
ping

Reply:
Let me fix the remaining issues and we'll see that after.